operators/src/adapters/raster_time.rs

```diff
@@ -0,0 +1,379 @@
+                            Some(0),
+                        )
+                        .unwrap(),
+                    },
+                    RasterTile2D {
+                        time: TimeInterval::new_unchecked(3, 6),
+                        tile_position: [0, 1].into(),
+                        global_geo_transform: Default::default(),
+                        grid_array: Grid::new(
+                            [3, 2].into(),
+                            vec![119, 120, 121, 122, 123, 124],
+                            Some(0),
+                        )
+                        .unwrap(),
+                    },
+                    RasterTile2D {
+                        time: TimeInterval::new_unchecked(6, 10),
+                        tile_position: [0, 0].into(),
+                        global_geo_transform: Default::default(),
+                        grid_array: Grid::new(
+                            [3, 2].into(),
+                            vec![125, 126, 127, 128, 129, 130],
+                            Some(0),
+                        )
+                        .unwrap(),
+                    },
+                    RasterTile2D {
+                        time: TimeInterval::new_unchecked(6, 10),
+                        tile_position: [0, 1].into(),
+                        global_geo_transform: Default::default(),
+                        grid_array: Grid::new(
+                            [3, 2].into(),
+                            vec![131, 132, 133, 134, 135, 136],
+                            Some(0),
+                        )
+                        .unwrap(),
+                    },
+                ],
+                result_descriptor: RasterResultDescriptor {
+                    data_type: RasterDataType::U8,
+                    spatial_reference: SpatialReference::wgs84().into(),
+                },
+            },
+        }
+        .boxed();
+
+        let thread_pool = ThreadPool::new(2);
+        let exe_ctx = ExecutionContext {
+            raster_data_root: Default::default(),
+            thread_pool: thread_pool.create_context(),
+        };
+        let query_rect = QueryRectangle {
+            bbox: BoundingBox2D::new_unchecked((0., 0.).into(), (1., 1.).into()),
+            time_interval: TimeInterval::new_unchecked(0, 10),
+            spatial_resolution: SpatialResolution::one(),
+        };
+        let query_ctx = QueryContext {
+            chunk_byte_size: 1024 * 1024,
+        };
+
+        let qp1 = mrs1
+            .initialize(&exe_ctx)
+            .unwrap()
+            .query_processor()
+            .unwrap()
+            .get_u8()
+            .unwrap();
+        // let s1 = qp1.query(query_rect, query_ctx).fuse();
+
+        let qp2 = mrs2
+            .initialize(&exe_ctx)
+            .unwrap()
+            .query_processor()
+            .unwrap()
+            .get_u8()
+            .unwrap();
+        // let s2 = qp2.query(query_rect, query_ctx).fuse();
```

Comment:
You could remove the commented-out code.

Reply:
ab11218997dc5eba42e7a3fd4d25034f54396ebe